README.md

```diff
@@ -131,23 +138,26 @@ Supported Platforms
 - type: `string`
 - description: Use web-based config for hostname and keyfile
 
-### herokuish_version
+### herokuish_version (deprecated)
 
-- default: `0.5.18`
+- default: `None`
 - type: `version`
-- description: The version of herokuish to install
+- description: The version of herokuish to install.
+Scheduled for deletion after 07/2021. Use `dokku_packages_state` instead.
 
-### plugn_version
+### plugn_version (deprecated)
 
-- default: `0.5.0`
+- default: `None`
 - type: `version`
-- description: The version of plugn to install
+- description: The version of plugn to install.
+Scheduled for deletion after 07/2021. Use `dokku_packages_state` instead.
 
-### sshcommand_version
+### sshcommand_version (deprecated)
 
-- default: `0.11.0`
+- default: `None`
```

Comment:
Is it the literal value `None` or an empty string? I think we need to be explicit here, maybe by adding the type?

Reply:
The literal should be `null` actually - the problem is that the `main.yml.base` (with default values) gets parsed in to a python dictionary, and then those values are inserted here.

I've now switched to an empty string, which makes things more explicit, and added some basic mapping between the python and yaml representation.